### ISSUE TITLE
Focus - define when focus gets applied and update components

### ISSUE THREAD

**Issue by thrbnhrtmnn** [deven-org/B01LER-Kitchen#751]
#### Description / User Story

For example in the checkbox component the focus ring already gets applied when pressing, but in design it is not there in pressed state. This behaviour should be checked again and if we decide to change it, it has to be changed in all components that have a focus ring. It is also happening at least in the Radio component.

----
#### Acceptance Criteria
- [ ] _Acceptance criteria 1_
- [ ] ...

----
#### Background information
- _Add relevant information here_
- ...
